_static/version_redirect.js

```diff
@@ -1,5 +1,11 @@
 var collator = new Intl.Collator(undefined, {numeric: true, sensitivity: 'base'});
 
+$.ajaxSetup({beforeSend: function (xhr) {
+    if (xhr.overrideMimeType) {
+        xhr.overrideMimeType("application/json");
+    }
+}});
```

Comment:
What's this? (Might deserve a comment)

Reply:
Trying to avoid "invalid XML" warnings on some browsers, notably Firefox.  Comment added.